modules/KalturaSupport/components/playlistAPI.js

```diff
@@ -190,6 +192,14 @@
 						'pager:pageSize': _this.getConfig('pageSize'),
 						'id': _this.playlistSet[_this.currentPlaylistIndex].id
 					};
+
+					// extend with paging props
+					$.extend(playlistRequest, {
```

Comment:
@vladm3 - this code is redundant if not using saved params (when creating a new request object).